src/analysis/retail/monk/mistweaver/modules/core/HotAttributor.ts

```diff
@@ -91,7 +91,9 @@ class HotAttributor extends Analyzer {
       //mists of life rem
       remDebug && this._newReMAttributionLogging(event, this.MistsOfLifeAttrib);
       this.hotTracker.addAttributionFromApply(this.MistsOfLifeAttrib, event);
-      this.hotTracker.hots[event.targetID][event.ability.guid].maxDuration = this.hotTracker._getDuration(this.hotTracker.hotInfo[event.ability.guid]);
+      this.hotTracker.hots[event.targetID][
```

Comment:
??

Reply:
uhhhh oops, cat on keyboard and it saved file lol